src/drawscene.cpp

```diff
@@ -90,18 +160,20 @@ void draw_anaglyph_3d_mode(Camera& camera, bool show_hud, Hud& hud,
 		draw_selectionbox(driver, hud, hilightboxes, show_hud);
 
 		if (draw_wield_tool)
-			camera.drawWieldedTool(&leftMove);
+			camera.drawWieldedTool(&movement);
 	}
 
 	guienv->drawAll();
 
 	//Right eye...
-	irr::core::vector3df rightEye;
-	irr::core::matrix4 rightMove;
-	rightMove.setTranslation(
-			irr::core::vector3df(g_settings->getFloat("3d_paralax_strength"),
-					0.0f, 0.0f));
-	rightEye = (startMatrix * rightMove).getTranslation();
+	calculate_3d_matrices(RIGHT,
+	                      camera,
+	                      halfInterocularDistance,
+	                      convergenceDistance,
+	                      projectionMatrix,
```

Comment:
And here too. Git doesn't like mixing tabs with spaces. use tabs only: one indentation level deeper than the `calculate_3d_matrices` call.